src/common/transformations/include/transformations/utils/gen_pattern.hpp

```diff
@@ -811,17 +813,20 @@ class GenericPattern : public ov::pass::pattern::op::Pattern {
                          pattern_value.get_index(),
                          "!=",
                          graph_value.get_index());
+            OPENVINO_LOG_GENPATTERN1(matcher, pattern_value, graph_value);
             return false;
         }
 
         auto value_node = graph_value.get_node_shared_ptr();
         if (!value_node->get_type_info().is_castable(m_type_info)) {
             _VERBOSE_LOG(level, "X OP type mismatch: ", m_signature, " vs ", graph_value);
+            OPENVINO_LOG_GENPATTERN2(matcher, pattern_value, graph_value);
```

Comment:
Just topic for future improvements, maybe is possible to merge these two log macros into one?